aspnetcore/zone-pivot-groups.yml

```diff
@@ -18,3 +18,15 @@ groups:
       title: Blazor Server
     - id: webassembly
       title: Blazor WebAssembly
+- id: blazor-hybrid-operating-systems
+  title: Operating System
+  prompt: Choose an operating system
```

Comment:
How about "Target Platform" or "Target Operating System" to distinguish between the host development machine operating system (Windows/Mac) vs the "target" (Windows/Mac/iOS/Android).